build.ps1

```diff
@@ -0,0 +1,35 @@
+[CmdletBinding()]
+param(
+    [Parameter()]
+    [ArgumentCompleter(
+        {
+            param($Command, $Parameter, $WordToComplete, $CommandAst, $FakeBoundParams)
+            $r = (Invoke-Build ?).Name
+
+            if ($WordToComplete) {
+                $r.Where{ $_ -match "^$WordToComplete" }
+            }
+            else {
+                $r
+            }
+        }
+    )]
+    [string]
+    $Task = '.'
+)
+
+if (-not (Get-PackageProvider NuGet -ErrorAction Ignore)) {
+    Write-Host "Installing NuGet package provider"
```

Comment:
I'm not familiar with what breakages would happen here honestly? If they already have nuget on the system, there's a good chance PowerShell already knows about it. If not, then yeah it's a possibility I suppose.

Reply:
It is the creation of an empty nuget.config file in the users top level NuGet folder that causes the problem, since it prevents the resolving of any packages when running nuget.exe or dotnet.exe. We have fixed this issue in Chocolatey, but the NuGet Package Provider is known still to cause the problem.

Reply:
> but the NuGet Package Provider is known still to cause the problem.

It looks like it was actually fixed in the NuGet Package Provider version `2.8.5.208` (at least I'm not seeing it on my test VM, and it looks like there was a PR: https://twitter.com/timheuer/status/1519357098601054208)

Reply:
Ooo, good to know. Last time I checked that PR, it hadn't been merged.